pkg/controller/kogitoruntime/kogitoruntime_controller.go

```diff
@@ -56,7 +58,16 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	err = c.Watch(&source.Kind{Type: &v1beta1.KogitoRuntime{}}, &handler.EnqueueRequestForObject{})
+	pred := predicate.Funcs{
```

Comment:
duplicated code, maybe move to some common package?

Reply:
I had moved `addFinalizer` and `handleFinalization` method to common package. Are you refering to same duplicate code?